### ISSUE TITLE
annotation, BoM, saving data

### ISSUE THREAD

**Issue by orangecms** [fiedka/fiedka#69]
We can parse file systems and already get a bunch of metadata that stems from the files themselves.

Now we can enrich the static information: Add a simple textarea for a comment per file, and add a button to save all the information for a start.
Then add a button to load back information.

Note: This involves work on the state management and rendering, which is currently not well optimized. 😅

**Comment by pietrushnic** [comment]
Why not to save that as repo or sth like that? When there will be comment, then instantly there is question about author of the comment to prove credibility.

**Comment by orangecms** [comment]
> Why not to save that as repo or sth like that? When there will be comment, then instantly there is question about author of the comment to prove credibility.

Yes, that is something I wanted to neglect deliberately for the time being.
Data requires infrastructure, and I think a repo could be good start indeed, great idea! 💡 Thank you :)

**Comment by orangecms** [comment]
Note: With #79, we have added first export functionalities now, so that you can export a list of UEFI components to remove as JSON and load it again.

**Comment by orangecms** [comment]
We should add an explanation of what SBoM is. Possible references:
https://blog.reversinglabs.com/blog/gartner-explains-why-sboms-are-critical-to-software-supply-chain-security-management
